### ISSUE TITLE
Add support to external .htmlhintrc file

### ISSUE THREAD

**Issue by fsmaia** [Microsoft/vscode-htmlhint#4]
As in "jscs.preset", it would be nice to have support for a external .htmlhintrc file. I see that putting the file directly in project is widely recommended, but there are some scenarios in which .htmlhintrc must be apart.

**Comment by badpenguin** [comment]
Yeah... its quite annoying the doctype error for ionic templates.
I've putted my .htmlhintrc in project's root but its not working.

**Comment by mike-kaufman** [comment]
> I've putted my .htmlhintrc in project's root but its not working.

@badpenguin  - can you please post some details of this.  E.g., a zip file of project that repros the behavior, and steps of what files you open.  Also, what folder do you open when you open code?

**Comment by mike-kaufman** [comment]
> As in "jscs.preset", it would be nice to have support for a external .htmlhintrc file.

I think the fix here is fairly straightforward - on [this line](https://github.com/Microsoft/vscode-htmlhint/blob/master/htmlhint-server/src/server.ts#L83), instead of treating `options` as a json object, treat it as a file path and load the options from that path.  You can then set the options from inside vscode.   Feel free to take a crack submit a PR!
